From 1be9e1c5a09b3645e47a82ca199bf7153b7ffc35 Mon Sep 17 00:00:00 2001
From: Mathieu Maret <mmaret@genymobile.com>
Date: Wed, 4 Nov 2015 15:21:55 +0100
Subject: [PATCH] arm:dt:msm8994 Correct regulator timming

Documentation describe slew-rate as "time in us it takes for the regulator to change votlage value in one step".
2s seems a lot and qpnp-lab-slew-rate is only 5000us.
So correct it as if a 1000 factor was added.

This reduce the kernel boot time of the karin windy by 80s
Tested on karin windy only

Signed-off-by: Mathieu Maret <mmaret@genymobile.com>
Signed-off-by: mydongistiny <jaysonedson@gmail.com>
Signed-off-by: Joe Maples <joe@frap129.org>
---
 arch/arm/boot/dts/qcom/msm-pmi8994.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/qcom/msm-pmi8994.dtsi b/arch/arm/boot/dts/qcom/msm-pmi8994.dtsi
index 1303ba6a27b..de8259a53a7 100644
--- a/arch/arm/boot/dts/qcom/msm-pmi8994.dtsi
+++ b/arch/arm/boot/dts/qcom/msm-pmi8994.dtsi
@@ -459,7 +459,7 @@
 
 				qcom,qpnp-ibb-min-voltage = <1400000>;
 				qcom,qpnp-ibb-step-size = <100000>;
-				qcom,qpnp-ibb-slew-rate = <2000000>;
+				qcom,qpnp-ibb-slew-rate = <2000>;
 				qcom,qpnp-ibb-use-default-voltage;
 				qcom,qpnp-ibb-init-voltage = <5500000>;
 				qcom,qpnp-ibb-init-amoled-voltage = <4000000>;
